zerver/webhooks/intercom/view.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from typing import Any, Dict, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('Intercom')
+@has_request_variables
+def api_intercom_webhook(request: HttpRequest, user_profile: UserProfile,
+                         payload: Dict[str, Any] = REQ(argument_type='body'),
+                         stream: Text = REQ(default='test')) -> HttpResponse:
+    topic = str(payload['topic'])
```

Comment:
I don't think you need to convert this to a `str`.

Reply:
I will change it. (run-mypy fails without it tho)